mongo/options/encryptoptions.go

```diff
@@ -16,39 +16,78 @@ const (
 	QueryTypeEquality string = "equality"
 )
 
-// RangeOptions specifies index options for a Queryable Encryption field supporting "rangePreview" queries.
-// Beta: The Range algorithm is experimental only. It is not intended for public use. It is subject to breaking changes.
+// RangeOptions specifies index options for a Queryable Encryption field
+// supporting "rangePreview" queries. Beta: The Range algorithm is experimental
+// only. It is not intended for public use. It is subject to breaking changes.
 type RangeOptions struct {
 	Min       *bson.RawValue
 	Max       *bson.RawValue
 	Sparsity  int64
 	Precision *int32
 }
 
-// EncryptOptions represents options to explicitly encrypt a value.
+// RangeOptionsBuilder contains options to configure Rangeopts for queryeable
+// encryption. Each option can be set through setter functions. See
+// documentation for each setter function for an explanation of the option.
+type RangeOptionsBuilder struct {
+	Opts []func(*RangeOptions) error
+}
+
+// Range creates a new RangeOptions instance.
+func Range() *RangeOptionsBuilder {
```

Comment:
These are unused and should be removed. We can add them in the future if they become necessary.

Reply:
RangeOptionsBuilder is used to create explicit options for FLE: https://github.com/mongodb/mongo-go-driver/blob/1b7cde6e7d946cc66da0c9ec57fcffc1a89e2877/mongo/client_encryption.go#L249

Reply:
Ah, I see where it's used now 👍 . Consider this comment resolved.